app/boards/shields/kyria/kyria_rev3.dtsi

```diff
@@ -29,13 +29,13 @@
 };
 
 &left_encoder {
-    resolution = <2>;
+    steps = <80>;
```

Comment:
I think steps or resolution don't need to be set here since the common dtsi sets them: https://github.com/zmkfirmware/zmk/blob/72d569ee155ef9b78269ed0813596d5d758c7482/app/boards/shields/kyria/kyria_common.dtsi#L24

Reply:
Fixed.